mlflow/R/mlflow/R/model.R

```diff
@@ -76,15 +76,19 @@ mlflow_predict_model <- function(model, data) {
 #' Loads an MLflow model. MLflow models can have multiple model flavors. Not all flavors / models
 #' can be loaded in R. This method by default searches for a flavor supported by R/MLflow.
 #'
-#' @param model_path Path to the MLflow model. The path is relative to the run with the given
-#'        run-id or local filesystem path without run-id.
-#' @param run_id Optional MLflow run-id. If supplied model will be fetched from MLflow tracking
-#'        server.
+#' @param model_uri The location, in URI format, of the MLflow model, for example:
+#'                  - ``file:///Users/me/path/to/local/model``
+#'                  - ``file:relative/path/to/local/model``
```

Comment:
`file:/` ? Whether `file:` should work is ambiguous based on the definition of URIs (so it depends on the implementation). The docs say to use a slash so we might as well be consistent (https://www.mlflow.org/docs/latest/tracking.html#where-runs-are-recorded)

Reply:
Hmm I read that if you want to have a relative path, you should use file:. I'll look up the source and put more info here.

Reply:
Talked offline. "file:" is the right scheme to use for relative paths. @sueann Do you think the comment is enough as is (it points file:relative/path/....) or do you think we should make it more explicit?